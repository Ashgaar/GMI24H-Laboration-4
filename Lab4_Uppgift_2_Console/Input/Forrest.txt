Run Forrest Run!
"Run Forrest run!"
'Run forrest run'
‘run forrest run’
“Run Forrest, Run!”
