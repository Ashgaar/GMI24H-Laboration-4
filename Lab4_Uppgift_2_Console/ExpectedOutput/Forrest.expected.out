forrest	5
run	10